src/App.tsx

```diff
@@ -1,26 +1,77 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import React, { useContext, useEffect, useMemo } from 'react';
+import { Header } from './Components/Header';
+import { TodoList } from './Components/TodoList';
+import { Todo } from './types/Todo';
+import { SortingTodos } from './types/Sorting';
+import { Footer } from './Components/Footer';
+import { DispatchContext, TodoContext } from './Components/TodoContext';
+import { getTodos } from './api/todos';
+import classNames from 'classnames';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const { dispatch, resetErrorMessage } = useContext(DispatchContext);
+  const { error } = useContext(TodoContext);
+
+  useEffect(() => {
+    getTodos()
+      .then(data => {
+        dispatch({ type: 'setTodos', payload: data });
+      })
+      .catch(() => {
+        dispatch({
+          type: 'setError',
+          payload: { errorMessage: 'Unable to load todos' },
+        });
+
+        resetErrorMessage();
+      });
+  }, []);
```

Comment:
Fix it 
<img width="1173" alt="image" src="https://github.com/mate-academy/react_todo-app-add-and-delete/assets/64750958/8ed315dd-a8f1-42e1-b369-b48f8df56eea">